setup.py

```diff
@@ -34,11 +34,9 @@
     "pytesseract==0.3.0",
     "pytest==5.1.2",
     "python-dateutil==2.8.0",
+    "python-xlib==0.25",
 ]
 
-if platform.system() == "Linux":
-    INSTALL_REQUIRES.append("xlib")
-
```

Comment:
From my understanding, this conditional import was necessary before to avoid a runtime exception for `xlib` on non-Linux systems. Will your code work cross-platform without raising an exception?

Reply:
this may have been the result of tinkering that I missed when committing